components/managedidentity/04-managed-identities.tf

```diff
@@ -1,23 +1,20 @@
-data "azurerm_resource_group" "cftsbox_intsvc" {
-  provider = azurerm.ptlsbox
-  name     = "managed-identities-cftsbox-intsvc-rg"
+data "azurerm_resource_group" "managed_identities" {
+  name     = "managed-identities-${local.mi_environment}-rg"
 }
 
-resource "azurerm_user_assigned_identity" "ptlsbox_managed_identity" {
-  provider = azurerm.ptlsbox
-
-  resource_group_name = data.azurerm_resource_group.cftsbox_intsvc.name
+resource "azurerm_user_assigned_identity" "managed_identity" {
+  resource_group_name = data.azurerm_resource_group.managed_identities.name
   location            = var.location
 
-  name = "monitoring-cftsbox-intsvc-mi"
+  name = "monitoring-${local.mi_environment}-mi"
 
   tags = local.common_tags
 }
 
 resource "azurerm_key_vault_access_policy" "ptlsbox_managed_identity_access_policy" {
```

Comment:
should this be inside the 05-keyvault.tf file as it is a KV resource or just merge the data lookup into this file?

Reply:
Is this a KV resource? I think the MI makes more sense to keep in the managed identities file. Unless you mean the access policy?